packages/core/src/behaviors.ts

```diff
@@ -172,6 +161,12 @@ export function spawnBehavior<TEvent extends EventObject, TEmitted>(
         }
       };
     },
+    start() {
+      mailbox.start();
```

Comment:
this most likely should also start a behavior or something, but I'm not sure about the relationships between different primitive types (as mentioned in another comment). All tests are passing at the moment - so at the very least this is under-specified and we should make sure those primitive types and communication protocols etc are well-defined and covered by tests.